packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -171,16 +172,25 @@ export const getCFDAccountDisplay = ({
 
 type TGetCFDAccount = TGetAccount & {
     is_eu?: boolean;
+    is_transfer_form?: boolean;
 };
 
-export const getCFDAccount = ({ market_type, sub_account_type, platform, is_eu }: TGetCFDAccount) => {
+export const getCFDAccount = ({
+    market_type,
+    sub_account_type,
+    platform,
+    is_eu,
+    is_transfer_form = false,
+}: TGetCFDAccount) => {
     let cfd_account_key = getCFDAccountKey({ market_type, sub_account_type, platform });
     if (!cfd_account_key) return undefined;
 
     if (cfd_account_key === 'financial' && is_eu) {
         cfd_account_key = 'cfd';
     }
 
+    if (cfd_account_key === 'ctrader' && is_transfer_form) return 'Ctrader';
```

Comment:
is this spelling correct or maybe it should be `cTrader`.
just curious

Reply:
@hamza-deriv this is correct, `Ctrader` is using for mapping to the correct icon which is 'IcBrandCtrader'